### ISSUE TITLE
Question about hyper-parameter search strategy

### ISSUE THREAD

**Issue by speedcell4** [pfnet/optuna#433]
Hi~
Generally, the neural network models perform differently with a different random seed. Thus even use the same hyper-parameters, models can still result in totally different scores. Does optuna take this into consideration? I mean, will optuna run with the same hyper-parameters several times and calculate the averaged scores? or just once for each hyper-parameter setting? and if the answer is the last one, is there any mechanism that I can force optuna to run with the first strategy?

**Comment by toshihikoyanase** [comment]
Thank you for your question. Currently, Optuna does not provide features to evaluate objective functions repeatedly.
For simple cases, I think you can use the following wrapper function:

```python
def objective(trial):
    ...

def objective_ntimes(trial):
    result = [objective(trial) for _ in range(5)]
    return np.nanmean(result)

study = optuna.create_study()
study.optimize(objective_ntimes, n_trials=100)
```

Note that this may increase computational time or may require special cares to apply pruning.

**Comment by speedcell4** [comment]
Thank you~
